SmartDeviceLink/SDLButtonName.h

```diff
@@ -100,4 +100,113 @@ extern SDLButtonName const SDLButtonNamePreset9;
 
 extern SDLButtonName const SDLButtonNameCustomButton;
 
+/**
+ * @abstract Represents the SEARCH button.
+ *
+ */
 extern SDLButtonName const SDLButtonNameSearch;
+
+
+#pragma mark - Climate Buttons
+
+/**
+ * @abstract Represents AC max button *
+ */
+extern SDLButtonName const SDLButtonNameAcMax;
```

Comment:
"Ac" here would be "AC" because it's an acronym. Acronyms should be all the same (upper or lower).